jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/LocalServerDiscovery.kt

```diff
@@ -108,7 +111,11 @@ public class LocalServerDiscovery(
 
 		// Try reading incoming messages but with a maximum
 		val foundServers = mutableSetOf<String>()
-		for (i in 0..maxServers) {
+
+		var i = 0
+		while (i < maxServers) {
+			i++
```

Comment:
If I'm not mistaken, this changes the behavior of the code.
Probably intended since it seems to have been broken before, but what's wrong with a `for (i in 0 until maxServers)` loop?

Alternatively, a `do {} while ()` loop could make sense here. I don't really like the suggested syntax xD

Reply:
Changed because of "Unused variable `i`.". Not too big a fan of the while either.

Reply:
I went back to the for loop and added a `@Suppress` annotation

Reply:
I see. Would a `repeat(n) {}` and `return@repeat` possibly work?

Reply:
Code uses both break and continue statements

Reply:
K, just suppressing the warning is fine.